modules/commerce_bluesnap_currency/commerce_bluesnap_currency.module

```diff
@@ -0,0 +1,86 @@
+<?php
+
+/**
+ * @file
+ * Provides Bluesnap currency resolver settings.
+ */
+
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_form_alter().
+ */
+function commerce_bluesnap_currency_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  // Alter commerce_currency_resolver_admin_conversion form and add
+  // bluesnap api settings.
+  if ($form_id == "commerce_currency_resolver_admin_conversion") {
+    // Get current settings.
+    $config = \Drupal::config('commerce_currency_resolver.currency_conversion');
+
+    $form['bluesnap'] = [
+      '#type' => 'details',
+      '#title' => t('Bluesnap Settings'),
+      '#open' => FALSE,
+      '#tree' => TRUE,
+    ];
+    $form['bluesnap']['username'] = [
+      '#type' => 'textfield',
+      '#title' => t('Username'),
+      '#default_value' => $config->get('bluesnap')['username'],
+    ];
+    $form['bluesnap']['password'] = [
+      '#type' => 'textfield',
+      '#title' => t('Password'),
+      '#default_value' => $config->get('bluesnap')['password'],
+    ];
+    $form['bluesnap']['mode'] = [
+      '#type' => 'radios',
+      '#title' => t('Mode'),
+      '#options' => [
+        'sandbox' => t('sandbox'),
+        'production' => t('Production'),
+      ],
+      '#default_value' => $config->get('bluesnap')['mode'],
+    ];
```

Comment:
**CHANGE REQUEST**

*Readability*

Leave an empty line to separate code into parts.